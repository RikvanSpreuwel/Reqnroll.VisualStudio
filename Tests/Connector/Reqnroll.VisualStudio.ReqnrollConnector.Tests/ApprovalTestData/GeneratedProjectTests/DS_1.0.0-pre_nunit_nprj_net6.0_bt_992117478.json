{
  "GeneratorOptions": {
    "FeatureFileCount": 1,
    "ScenarioPerFeatureFileCount": 1,
    "NewProjectFormat": true,
    "TargetFramework": "net6.0",
    "ReqnrollPackageVersion": "1.0.0-pre20240125-60"
  }
}
