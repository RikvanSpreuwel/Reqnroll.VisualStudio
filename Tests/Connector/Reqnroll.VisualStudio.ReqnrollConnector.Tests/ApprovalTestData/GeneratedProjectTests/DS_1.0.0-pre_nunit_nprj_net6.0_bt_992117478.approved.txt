{
  "stepDefinitions": [],
  "sourceFiles": {},
  "typeNames": {},
  "analyticsProperties": {
    "Connector": "<connector>",
    "ImageRuntimeVersion": "v4.0.30319",
    "SFFile": "Reqnroll.dll",
    "SFFileVersion": "1.0.0",
    "SFProductVersion": "1.0.0+c0877c79ce29cc34ebd04fd85ea035a18b1874b8",
    "SourcePaths": "1",
    "StepDefinitions": "3",
    "TargetFramework": ".NETCoreApp,Version=v6.0",
    "TypeNames": "0"
  }
}